gammagl/mpops/torch.py

```diff
@@ -297,5 +297,9 @@ def gspmm(index, weight, x, reduce='sum'):
     """
     if reduce == 'sum':
         return c_spmm_sum(index, weight, x)
+    elif reduce == 'mean':
+        return c_spmm_mean(index, weight, x)
+    elif reduce == 'max':
+        return c_spmm_max(index, weight, x)
     else:
         raise Exception("Unsupported reduce type, please choose from ['sum', ].")
```

Comment:
This should be changed into: raise Exception("Unsupported reduce type, please choose from ['sum', 'mean', 'max'].")